samples/cpp/image_generation/text2image_concurrency.cpp

```diff
@@ -22,7 +22,23 @@ int32_t main(int32_t argc, char* argv[]) try {
         prompts.push_back(argv[i]);
 
     // Prepare initial pipeline and compiled models into device
-    pipelines.emplace_back(models_path, device);
+    ov::AnyMap properties;
+    if (device == "NPU") {
```

Comment:
I think we need the same for img2img_concurrency sample

Reply:
done